packages/flare/tests/python/test_ingest_events.py

```diff
@@ -0,0 +1,193 @@
+
+    api_key, tenant_id = get_api_credentials(app=app)
+    assert api_key == "some_api_key"
+    assert tenant_id == 11111
+
+
+def test_fetch_and_print_feed_results_expect_exception() -> None:
+    logger = MagicMock()
+    app = MagicMock()
+    with patch.object(
+        app.service,
+        "confs",
+        {
+            "flare": {
+                "endpoints": {
+                    "me_feed_endpoint": "https://api.flare.io/firework/api2/me/feed"
+                }
+            }
+        },
+    ):
+        fetch_and_print_feed_results(
+            logger=logger, app=app, api_key="some_key", tenant_id=11111
+        )
+
+    logger.error.assert_called_once_with("Exception=Failed to fetch API Token")
+
+
+def test_fetch_and_print_feed_results_expect_non_200_response() -> None:
+    logger = MagicMock()
+    app = MagicMock()
+
+    with patch("cron_job_ingest_events.FlareAPI") as MockFlareAPI:
+        response_mock = Mock()
+        type(response_mock).status_code = PropertyMock(return_value=400)
+        type(response_mock).text = PropertyMock(return_value="Bad Request")
+
+        flare_api_mock_instance = MockFlareAPI.return_value
+        flare_api_mock_instance.retrieve_feed.return_value = [response_mock]
+
+        with patch("time.sleep", return_value=None):
+            with patch.object(
+                app.service,
+                "confs",
+                {
+                    "flare": {
+                        "endpoints": {
+                            "me_feed_endpoint": "https://api.flare.io/firework/api2/me/feed"
+                        }
+                    }
+                },
+            ):
+                fetch_and_print_feed_results(
+                    logger=logger, app=app, api_key="some_key", tenant_id=11111
+                )
+
+    logger.error.assert_called_once_with("Bad Request")
+
+
+def test_fetch_and_print_feed_results_expect_feed_response(capfd: Any) -> None:
+    logger = MagicMock()
+    app = MagicMock()
+
+    with patch("cron_job_ingest_events.FlareAPI") as MockFlareAPI:
+        response_mock = Mock()
+        type(response_mock).status_code = PropertyMock(return_value=200)
+        response_mock.json.return_value = {
+            "next": "some_next_value",
+            "items": [
+                {
+                    "actor": "this guy",
+                },
+                {
+                    "actor": "some other guy",
+                },
+            ],
+        }
+
+        flare_api_mock_instance = MockFlareAPI.return_value
+        flare_api_mock_instance.retrieve_feed.return_value = [response_mock]
+
+        with patch("time.sleep", return_value=None):
+            with patch.object(
+                app.service,
+                "confs",
+                {
+                    "flare": {
+                        "endpoints": {
+                            "me_feed_endpoint": "https://api.flare.io/firework/api2/me/feed"
```

Comment:
You have to mock a response from a request since it's not idea to use or get real data. So this is just exercising the code following the response validating that is working as we expect it to.